cip-drep-metadata/README.md

```diff
@@ -0,0 +1,248 @@
+
+## Specification
+CIP-1694 specifies that metadata anchors are optional for DRep registration and updates. This CIP covers metadata for the aforementioned transaction types, but it does not cover metadata for voting transactions. 
+
+#### A Note on Teams
+CIP-1694 allows for DReps to be registered using a native or Plutus script credential, this implies that individuals could organise to form a team that would have a broad range of experitise, and would perhaps therefore be more attractive to some delegating Ada Holders. 
+
+Participants at the workshop held to debut the first draft of this CIP strongly advocated in favour of a CIP that would include features that would assist a DRep comprised of a team of individuals properly describe their endeavour to delegating Ada Holders and anyone else reading the metadata associated with their registration.
+
+This CIP has not included these features, the decision not to include these features was made in order to simplify this CIP so that it became suitable for the minimum viable level of tooling, with the expectation that further CIPs will build on it. 
+
+### Formatting
+If DRep Metadata is not formatted in the way described by this CIP then tooling SHOULD either:
+1. Deal with it as described in CIP-100 (by rendering the raw unformatted document)
+2. Not display any of the metadata
+
+In either case the tooling SHOULD provide a warning that this has happened and link to the url described in the metadata anchor of the DRep registration.
+
+### Missing
+If the DRep metadata cannot be found at the address specified in the anchor then the tooling SHOULD provide a warning. There is no need to link to the url in the metadata anchor. 
+
+### Witnesses
+DRep Metadata will not follow the CIP-100 specification related to signing the metadata, the [authors property](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0100/README.md#high-level-description) can be left blank without the need for tooling providers to warn their users that the author has not been validated. Instead the author can be derived from the DRep ID associated with the registration. The need for an `authors` field will also be discarded in favour of including usernames inside of the `body` field. For the avoidance of doubt this CIP recommends that the entire authors property be left blank, and that tooling ignore it. 
+
+### Extended Vocabulary
+Like CIP-108, this CIP also extends the potential vocabulary of CIP-100's `body` property. 
+
+>**Note** For all of these properties tooling providers will need to be aware that they are responsible for what they display to their users and that these fields could be used for illegal, unsavoury, or innapropriate purposes or language. Therefore they will be responsible for having a terms of service which they enforce to moderate what they show to their users.
```

Comment:
```suggestion
>**Note** For all of these properties tooling providers will need to be aware that they are responsible for what they display to their users and that these fields could be used for illegal, unsavoury, or inappropriate purposes or language.
```
remove part of enforcement, just leave it as a warning to implementors?
cc @Crypto2099 @rphair

Reply:
I think that this note should be changed to a reminder for tooling providers, to remind them that when they display information from DRep metadata that they will be displaying user generated content. And that for whatever reason, they may decide that they do not want to display certain user generated content to their other users. Therefore they may also need to introduce a TOS and some form of moderation to handle this.

Reply:
I say this because its something that I easily forget and I have noticed this forgetfulness in other people also.

Reply:
thanks @Thomas-Upfield - there are three parts to this proposition, so I wanted to address all of them individually:
> a reminder for tooling providers, to remind them that when they display information from DRep metadata that they will be displaying user generated content.

👍 1) This is correct and sufficient: and also well-precented in the security field in considering information "tainted" whenever it is entered by a user.  Implementors, users, and potential regulators will be able to determine rights and responsibilities form this statement alone.

> And that for whatever reason, they may decide that they do not want to display certain user generated content to their other users.

👎 2) This is not necessary and leads immediately to the same red flags we pointed out at the meeting today: "condescension" I think we called it, and a suggestion of "nannying" that is considered obligatory according to one cultural bias but unacceptable in another.

If we suggest the censorship, it will be considered an _recommendation_ or at least an _endorsement_ of censorship, as well as an invitation to those potential regulators... so the "disclaimer" does not actually disclaim anything and will in fact be seen as an acceptance of liability that we wouldn't otherwise have.

> Therefore they may also need to introduce a TOS and some form of moderation to handle this.

👎 3) Also unnecessary for the same reason.  Making point #1 above is enough: to say that the tooling providers are posting user data to the blockchain.  They may "need" moderation and may not, and nobody can anticipate what their jurisdiction or legal precedent might be.  If "we" tell them what to do, "we" become _responsible_ for what they do: and the "potential regulators" are suddenly interested in who "we" are.

To everyone else, the presumption of a hypothetical censor _creates_ this censor: a centralised component in an otherwise decentralised process.  We need to say as little as possible about the moderation issue and simply tell tooling providers they are conveying / displaying / propagating user generated content.  Each tooling provider will have their own notion of liability and we cannot afford for this to be either our technical or our legal concern.

Reply:
... cross-referencing the related review points (and hoping further responses can continue in a single thread here, then resolved in those threads when resolved here):
https://github.com/cardano-foundation/CIPs/pull/788#discussion_r1533301310
https://github.com/cardano-foundation/CIPs/pull/788#discussion_r1548122679
https://github.com/cardano-foundation/CIPs/pull/788#discussion_r1546389557
https://github.com/cardano-foundation/CIPs/pull/788#discussion_r1546394333